src/validation.ts

```diff
@@ -35,6 +35,25 @@ export function isValidAccountAddress(address: string): boolean {
     }
 }
 
+const MAX_UINT_64 = 18446744073709551615n; // 2^64 - 1
+
+/**
+ * Checks whether the amount is an integer within the allowed range of a unsigned 64-bit integer.
+ * @param x the number to validate
+ * @returns true if the number is valid, otherwise false
+ */
+export function isValidUInt64(x: bigint | number): boolean {
```

Comment:
This method fails for the GetInstanceInfo on input that should be valid. This is because the json-bigint library uses are non-native bigint implementation. So we have to update `app.ts` with:
```javascript
JSON.parse = JSONbig({ useNativeBigInt: true }).parse;
JSON.stringify = JSONbig({ useNativeBigInt: true }).stringify;
```
then it seems to work as expected.